maat-court-data-api/src/main/java/gov/uk/courtdata/eform/dto/EformStagingDTO.java

```diff
@@ -0,0 +1,18 @@
+package gov.uk.courtdata.eform.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.io.Serializable;
+
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+public class EformStagingDTO implements Serializable {
+    
+    private Integer usn;
+    private String type;
```

Comment:
@mtac50 Can you confirm that this service should support the MAAT_REF field?

Reply:
yes this was an oversight on my part. It'll be useful to have the `maat_ref` so that we can see if an application has already been created for a given `usn`

Reply:
ok this has been added now